components/BottomSheet/src/MDCBottomSheetController.h

```diff
@@ -0,0 +1,66 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+@protocol MDCBottomSheetControllerDelegate;
+
+/**
+ A view controller for presenting other view controllers as bottom sheets.
+
+ https://material.io/guidelines/components/bottom-sheets.html
+
+ Show a bottom sheet by creating an MDCBottomSheetController instance with a contentViewController
+ and presenting it with -[UIViewController presentViewController:animated:completion].
+ MDCBottomSheetController automatically sets the appropriate presentation style and
+ transitioningDelegate for the bottom sheet behavior.
+ */
+@interface MDCBottomSheetController : UIViewController
+
+/**
+ The view controller being presented as a bottom sheet.
+ */
+@property(nonatomic, strong, nonnull, readonly) UIViewController *contentViewController;
+
+/**
+ The bottom sheet delegate.
+ */
+@property(nonatomic, weak, nullable) id<MDCBottomSheetControllerDelegate> delegate;
+
+/**
+ Initializes the controller with a content view controller.
+
+ @param contentViewController The view controller to be presented as a bottom sheet.
+ */
+- (nonnull instancetype)initWithContentViewController:
```

Comment:
If anything using a wrapper view controller would make it easier to use bottom sheets in storyboards. The API would need to be changed to allow assignment to contentViewController after instantiation but otherwise it's fine. 

Is it possible to assign a presentationController in storyboards?